ibm/service/scc/resource_ibm_scc_instance_settings.go

```diff
@@ -34,8 +34,20 @@ func ResourceIbmSccInstanceSettings() *schema.Resource {
 						"instance_crn": &schema.Schema{
 							Type:        schema.TypeString,
 							Optional:    true,
+							Computed:    true,
 							Description: "The Event Notifications instance CRN.",
 						},
+						"source_description": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Computed:    true,
+							Description: "The description of the source in Event Notifications connected Security and Compliance Center",
+						},
+						"source_name": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The name of the Event Notifications source connected Security and Compliance Center instance CRN.",
```

Comment:
Although its marked as optional, this should probably say its required if a value is passed for `instance_crn`. Otherwise you get this:

```
│ Error: event_notifications.source_name needs to be defined along with event_notifications.instance_crn
│ 
│   with module.create_scc_instance.ibm_scc_instance_settings.scc_instance_settings[0],
│   on ../../main.tf line 89, in resource "ibm_scc_instance_settings" "scc_instance_settings":
│   89: resource "ibm_scc_instance_settings" "scc_instance_settings" {
```